test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] The essence of git ignoring this file is that it shouldn't appear in the remote repository (it should not appear in Github). I can see that you have git ignored this file which is the first step. To complete the process, kindly follow the instructions in this stack overflow article https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore. 

#### You should replace the _**some-directory**_ with the name of this file in the first command. Remember to git add and git commit after running the first command in that article before pushing it. After you push it, kindly check the remote repository that this file no longer exists before requesting a review. 👍 💻

Reply:
So should I first do it on my local machine then when I push it , it will be ignored (test.md file)

Reply:
Thanks for the review, I've finally done it.

Reply:
@zdnahom , yes, you should run those commands on the local machine